statsmodels/sandbox/distributions/tests/test_norm_expan.py

```diff
@@ -20,10 +21,12 @@
 
 class CheckDistribution(object):
 
-    def test_smoke(self):
+    @pytest.mark.smoke
+    def test_dist1(self):
         self.dist1.rvs(size=10)
         self.dist1.pdf(np.linspace(-4, 4, 11))
 
+    def test_dist2_roundtrip(self):
```

Comment:
splitting this one is fine, 
name should be `test_cdf_ppf_roundtrip` to be more informative

Reply:
Change this test name, and rebase